code/modules/modular_computers/file_system/programs/maintenance/themes.dm

```diff
@@ -25,29 +33,43 @@
 	return TRUE
 
 ///Called post-installation of an application in a computer, after 'computer' var is set.
-/datum/computer_file/program/maintenance/theme/on_install()
```

Comment:
I lack a better place to comment this, in testing the default theme menu is showing this dead click. Doesn't toggle a new theme, but it does appear even without a data disc.
<img width="396" height="600" alt="image" src="https://github.com/user-attachments/assets/fbe6043a-1a8b-40b9-97f1-98bc5792736a" />